cmd/podman/common/netflags.go

```diff
@@ -84,15 +88,62 @@ func DefineNetFlags(cmd *cobra.Command) {
 	)
 }
 
+// GenerateTempNetFlags is how we convert the containerCLIopts from the libpod endpoint, into
+// net flags which are then converted to net options.
+func GenerateTempNetFlags(net *entities.NetFlags, infra bool) (pflag.FlagSet, error) {
```

Comment:
What is this needed for? The use of reflection here is making me a bit cautious...

Reply:
Yes I don't think this is a good solution. The server side should never touch flags.

Reply:
Basically this is a conduit for converting between podcreateoptions and containercreateoptions on the libpod handler end, the issue with any other approach is that more often than not it breaks everything else.

Reply:
This all stems from the switch from pod specgen to create options. This switch "upwards" in the stack causes the input to change a bit